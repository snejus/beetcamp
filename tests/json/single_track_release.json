{
  "@context": "https://schema.org",
  "@id": "https://mega-tech.bandcamp.com/track/arangel",
  "@type": "MusicRecording",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "track_id",
      "value": 2868806546
    },
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 919724444
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "all_rights_reserved"
    }
  ],
  "byArtist": {
    "@type": "MusicGroup",
    "name": "Matriark"
  },
  "copyrightNotice": "All Rights Reserved",
  "dateModified": "01 Jun 2023 17:03:15 GMT",
  "datePublished": "09 Nov 2020 00:00:00 GMT",
  "description": "mt004\n\nmastering by alexander salomonsen\n\n\nI'm very happy to announce Andrea's debut track. <3\n\nhttps://soundcloud.com/matriarkcph\nhttps://www.instagram.com/andreaapettersen/",
  "duration": "P00H07M01S",
  "image": "https://f4.bcbits.com/img/a0919724444_10.jpg",
  "inAlbum": {
    "@type": "MusicAlbum",
    "albumRelease": [
      {
        "@id": "https://mega-tech.bandcamp.com/track/arangel",
        "@type": [
          "MusicRelease",
          "Product"
        ],
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "item_id",
            "value": 2868806546
          },
          {
            "@type": "PropertyValue",
            "name": "item_type",
            "value": "t"
          },
          {
            "@type": "PropertyValue",
            "name": "selling_band_id",
            "value": 563736578
          },
          {
            "@type": "PropertyValue",
            "name": "type_name",
            "value": "Digital"
          },
          {
            "@type": "PropertyValue",
            "name": "art_id",
            "value": 919724444
          }
        ],
        "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
        "image": [
          "https://f4.bcbits.com/img/a0919724444_10.jpg"
        ],
        "musicReleaseFormat": "DigitalFormat",
        "name": "Arangel",
        "offers": {
          "@type": "Offer",
          "availability": "OnlineOnly",
          "price": 2.0,
          "priceCurrency": "EUR",
          "priceSpecification": {
            "minPrice": 2.0
          },
          "url": "https://mega-tech.bandcamp.com/track/arangel#t2868806546-buy"
        }
      }
    ],
    "albumReleaseType": "SingleRelease",
    "name": "Arangel",
    "numTracks": 1
  },
  "keywords": [
    "Malmö"
  ],
  "mainEntityOfPage": "https://mega-tech.bandcamp.com/track/arangel",
  "name": "Arangel",
  "publisher": {
    "@id": "https://mega-tech.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 563736578
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "has_policies",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "image_height",
        "value": 364
      },
      {
        "@type": "PropertyValue",
        "name": "image_id",
        "value": 32474138
      },
      {
        "@type": "PropertyValue",
        "name": "image_width",
        "value": 640
      }
    ],
    "description": "Malmø/Cph label run by Megatech and Matriark",
    "foundingLocation": {
      "@type": "Place",
      "name": "Malmö, Sweden"
    },
    "image": "https://f4.bcbits.com/img/0032474138_10.jpg",
    "name": "Megatech Industries",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "a"
          }
        ],
        "name": "artists",
        "url": "https://mega-tech.bandcamp.com/artists"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "music",
        "url": "https://mega-tech.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "p"
          }
        ],
        "name": "merch",
        "url": "https://mega-tech.bandcamp.com/merch"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "community",
        "url": "https://mega-tech.bandcamp.com/community"
      }
    ]
  }
}
