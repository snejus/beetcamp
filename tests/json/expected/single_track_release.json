{
  "album": null,
  "albumtype": "single",
  "albumtypes": ["single"],
  "arranger": null,
  "artist": "Matriark",
  "artist_credit": null,
  "artist_id": "https://mega-tech.bandcamp.com",
  "artist_sort": null,
  "artists": ["Matriark"],
  "artists_credit": [],
  "artists_ids": ["https://mega-tech.bandcamp.com"],
  "artists_sort": [],
  "barcode": null,
  "bpm": null,
  "catalognum": "mt004",
  "comments": "mt004\n\nmastering by alexander salomonsen\n\n\nI'm very happy to announce Andrea's debut track. <3\n\nhttps://soundcloud.com/matriarkcph\nhttps://www.instagram.com/andreaapettersen/",
  "composer": null,
  "composer_sort": null,
  "country": "SE",
  "data_source": "bandcamp",
  "data_url": "https://mega-tech.bandcamp.com/track/arangel",
  "day": 9,
  "disctitle": null,
  "genre": null,
  "index": null,
  "initial_key": null,
  "label": "Megatech Industries",
  "length": 421,
  "lyricist": null,
  "mb_workid": null,
  "media": "Digital Media",
  "medium": null,
  "medium_index": null,
  "medium_total": null,
  "month": 11,
  "release_track_id": null,
  "release_group_title": null,
  "style": null,
  "title": "Arangel",
  "track_alt": null,
  "track_id": "https://mega-tech.bandcamp.com/track/arangel",
  "work": null,
  "work_disambig": null,
  "year": 2020
}
